be/src/exprs/bitmap_function.cpp

```diff
@@ -302,6 +302,31 @@ void BitmapFunctions::bitmap_union(FunctionContext* ctx, const StringVal& src, S
     }
 }
 
+// this is the read init function for bitmap_intersect
+void BitmapFunctions::bitmap_intersect_init_real(FunctionContext* ctx, StringVal* dst) {
+    dst->is_null = true;
```

Comment:
why initial result bitmap as null? it seems that it will return empty bitmap when result bitmap is null

Reply:
The initial result bitmap must be null. Otherwise, the intersection between dst and src will be empty.